.ci_support/linux_64_.yaml

```diff
@@ -3,6 +3,6 @@ cdt_name:
 channel_sources:
 - conda-forge
 channel_targets:
-- conda-forge e3sm_dev
+- conda-forge e3sm_diags_dev
```

Comment:
Hey @xylar, let me know if this is correct. I'll update my [guide](https://acme-climate.atlassian.net/wiki/spaces/IPD/pages/3616735236/Releasing+E3SM+Software+on+Anaconda+conda-forge+channel) on Confluence with these instructions.

Reply:
I'm confident this is right based on the [release candidate](https://github.com/conda-forge/e3sm_to_cmip-feedstock/pull/27) I made for e3sm_to_cmip. I'll move forward with merging this PR.

Reply:
@tomvothecoder, sorry, I missed this.  You shouldn't edit most files in `.ci_support` directly because a rerender will just change them back.  This needs to be set in `recipe/conda_build_config.yaml`:
https://github.com/conda-forge/e3sm_diags-feedstock/pull/25/files#diff-ff61408cdc05bc9667deeadb55e4aaceb1371972076b6bf6934f9008920f2bd2

Reply:
I see, thanks for catching my mistake!

I used the e3sm_to_cmip PR above as a guide and saw `linux_64_.yaml` updated, so I incorrectly updated here too.  If I understand correctly, the rerender updated that file based on our change to `recipe/conda_build_config.yaml`' (they're linked together).

I'll document this on the Confluence guide.

Reply:
Yes, that's exactly right.